src/components/views/context_menus/DeviceContextMenu.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2021 Šimon Brandner <simon.bra.ag@gmail.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { useEffect, useState } from "react";
+
+import MediaDeviceHandler, { MediaDeviceKindEnum } from "../../../MediaDeviceHandler";
+import IconizedContextMenu, { IconizedContextMenuOptionList, IconizedContextMenuRadio } from "./IconizedContextMenu";
+import { IProps as IContextMenuProps } from "../../structures/ContextMenu";
+import { _t, _td } from "../../../languageHandler";
+
+const SECTION_NAMES: Record<MediaDeviceKindEnum, string> = {
+    [MediaDeviceKindEnum.AudioInput]: _td("Input devices"),
+    [MediaDeviceKindEnum.AudioOutput]: _td("Output devices"),
+    [MediaDeviceKindEnum.VideoInput]: _td("Cameras"),
```

Comment:
Looks like this probably should be `VideoOutput` instead

Reply:
`VideoOutput`? This is quite surely `VideoInput` since a camera is an input...

Reply:
Absolutely, I got confused for a second there

Reply:
No worries